src/items/implementations.md

```diff
@@ -174,6 +174,15 @@ be instantiable with the same set of types for the input type parameters. -->
 r[items.impl.trait.orphan-rule]
 #### Orphan rules
 
+r[items.impl.trait.orphan-rule.rationale]
+The orphan rule helps ensure that other people's code can't break your code, and vice versa.
+If an external crate implements an external trait for an external type, and your crate also
+implements the same trait for the same type, the compiler wouldn't know which implementation
+to use.\
+The orphan rule prevents this by requiring that either the trait or some type in the
+implementation is local to your crate, ensuring only one crate defines the implementation and
+thereby maintaining coherence.
```

Comment:
I would personally put a “that” here:

```suggestion
implementation is local to your crate, ensuring that only one crate defines the implementation
and thereby maintaining coherence.
```

Reply:
Probably we'll want to move this into a note section, but it's generally going in the right direction.  The one caveat I'd add is that the problem being prevented here isn't really about the compiler not knowing what to pick.  The compiler knows what to do about that.  It'd throw an ambiguity error.  But what that means is that adding an impl for your own types or of your own traits would be breaking.  So it's mostly about saving space, and that's the real rationale.  Of course, whenever you talk about removing the rule, you do find out all the ways that we are leaning on the rule more technically, and those reasons are a bit more subtle.  E.g.:

- https://github.com/rust-lang/rust/issues/136979